### PR TITLE
Mk2 expansion

### PR DESCRIPTION
**Pull request by xFluffyDemon**
MK2 expansion configs, missing engines and some misc stuff like rcs pods, nuke reactor

### CODE REVIEW

File: GameData/RealismOverhaul/RO_SuggestedMods/MK2_Expansion/MK2_FuelTank.cfg [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+        volume = 8000
+        type = Fuselage
+    }
+}
+
+@PART[M2X_Short25adapter]:FOR[RealismOverhaul]
+{
+    %RSSROConfig = True
+    %rescaleFactor = 1.722222
+    @mass = 0.774
```
Comment (Capkirk123) [inline GameData/RealismOverhaul/RO_SuggestedMods/MK2_Expansion/MK2_FuelTank.cfg:131]: Seems too light, even lighter than the standard short fuselage despite being larger

File: GameData/RealismOverhaul/RO_SuggestedMods/MK2_Expansion/MK2_Aero.cfg [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+	%skinMaxTemp = 2500
+}
+
+@PART[M2X_Tailboom]:FOR[RealismOverhaul]
+{
+    %RSSROConfig = True
+    %rescaleFactor = 1.722222
+    @mass = 0.915
+}
+
```
Comment (Capkirk123) [inline GameData/RealismOverhaul/RO_SuggestedMods/MK2_Expansion/MK2_Aero.cfg:12]: These intakes seem to end up with two intake modules ingame. Any stock modules and resources should be removed

File: GameData/RealismOverhaul/RO_SuggestedMods/MK2_Expansion/MK2_Structural.cfg [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+    %RSSROConfig = True
+    %rescaleFactor = 1.722222
+    @mass = 0.2
+}
+
+@PART[M2X_Endcap]:FOR[RealismOverhaul]
+{
+    %RSSROConfig = True
+    %rescaleFactor = 1.722222
+    @mass = 1
```
Comment (Capkirk123) [inline GameData/RealismOverhaul/RO_SuggestedMods/MK2_Expansion/MK2_Structural.cfg:25]: Seems a little heavy for what it is.

File: GameData/RealismOverhaul/RO_SuggestedMods/MK2_Expansion/MK2_Utility.cfg [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+		%captureMaxRvel = 0.1 // 0.3
+		%captureMinFwdDot = 0.998
+		%captureMinRollDot = -3.40282347E+38
+		%captureRange = 0.05 // 0.06
+		%minDistanceToReEngage = 0.25 // 1.0
+		%undockEjectionForce = 0.1 // 10
+	}
+}
+
+@PART[M2X_Mk2BatteryBank]:FOR[RealismOverhaul]
```
Comment (Capkirk123) [inline GameData/RealismOverhaul/RO_SuggestedMods/MK2_Expansion/MK2_Utility.cfg:35]: Only holding 2400 EC isn't much. This should probably just be configured as a ServiceModule, or not configured at all.

File: GameData/RealismOverhaul/RO_SuggestedMods/MK2_Expansion/MK2_Utility.cfg [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+    %RSSROConfig = True
+    %rescaleFactor = 1.722222
+    @mass = 0.550
+}
+
+@PART[M2X_ShieldedDockingPort]:FOR[RealismOverhaul]
+{
+    %RSSROConfig = True
+    %rescaleFactor = 1.722222
+    @mass = 0.5
```
Comment (Capkirk123) [inline GameData/RealismOverhaul/RO_SuggestedMods/MK2_Expansion/MK2_Utility.cfg:53]: This should be heavier than the unshielded docking port (or the shielded docking port lighter)

File: GameData/RealismOverhaul/RO_SuggestedMods/MK2_Expansion/MK2_Utility.cfg [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+}
+
+@PART[M2X_Nosebay]:FOR[RealismOverhaul]
+{
+    %RSSROConfig = True
+    %rescaleFactor = 1.722222
+    @mass = 0.5
+}
+
+@PART[M2X_PoddedRadiator]:FOR[RealismOverhaul]
```
Comment (Capkirk123) [inline GameData/RealismOverhaul/RO_SuggestedMods/MK2_Expansion/MK2_Utility.cfg:92]: This is absurdly powerful as compared to existing radiators, it's cooling capacity should be decreased.

File: GameData/RealismOverhaul/RO_SuggestedMods/MK2_Expansion/MK2_Utility.cfg [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+{
+    %RSSROConfig = True
+}
+
+@PART[M2X_SolarpanelPod]:FOR[RealismOverhaul]
```
Comment (Capkirk123) [inline GameData/RealismOverhaul/RO_SuggestedMods/MK2_Expansion/MK2_Utility.cfg:97]: These solar panels seem too powerful for their size.

File: GameData/RealismOverhaul/RO_SuggestedMods/MK2_Expansion/MK2_Aero.cfg [hunk 8]
```diff
@@ -0,0 +1,184 @@
...
+    {
+        name = ModuleFuelTanks
+        volume = 1500
+        type = Fuselage
+    }
+
+	!MODULE[ModuleResourceIntake] {}
+	!RESOURCE[IntakeAir] {}
+
+	MODULE
```
Comment (Capkirk123) [inline GameData/RealismOverhaul/RO_SuggestedMods/MK2_Expansion/MK2_Aero.cfg:130]: This seems incredibly powerful compared to other radiators ingame. Also, precoolers don't do anything with AJE.

File: GameData/RealismOverhaul/RO_SuggestedMods/Squad/RO_Squad_Command.cfg [hunk 9]
```diff
@@ -1,1372 +0,0 @@
-// Probe directory:
```
[the remark this answers is not in the document]
Comment (xFluffyDemon) [reply GameData/RealismOverhaul/RO_SuggestedMods/Squad/RO_Squad_Command.cfg:1]: yes, shouldn't have been there to begin with

[the remark this answers is not in the document]
Comment (Capkirk123) [reply GameData/RealismOverhaul/RO_SuggestedMods/Squad/RO_Squad_Command.cfg:1]: No, this should not be deleted, it's part of the config for squad parts.

File: GameData/RealismOverhaul/RO_SuggestedMods/MK2_Expansion/MK2_Aero.cfg (hunk 2 above)
Comment (Capkirk123) [reply GameData/RealismOverhaul/RO_SuggestedMods/MK2_Expansion/MK2_Aero.cfg:12]: They are not, a bunch of stock modules end up still on the parts. These should be removed.

File: GameData/RealismOverhaul/RO_SuggestedMods/MK2_Expansion/MK2_Utility.cfg (hunk 4 above)
Comment (Capkirk123) [reply GameData/RealismOverhaul/RO_SuggestedMods/MK2_Expansion/MK2_Utility.cfg:35]: That's not what I suggested.

File: GameData/RealismOverhaul/RO_SuggestedMods/Squad/RO_Squad_Command.cfg [hunk 10]
```diff
@@ -530,7 +530,10 @@
 
 	@MODULE[ModulePartVariants]
 	{
```
Comment (Capkirk123) [inline GameData/RealismOverhaul/RO_SuggestedMods/Squad/RO_Squad_Command.cfg:532]: I don't think you meant to put this here

Comment (Capkirk123) [reply GameData/RealismOverhaul/RO_SuggestedMods/Squad/RO_Squad_Command.cfg]: This is still being changed

File: GameData/RealismOverhaul/RO_SuggestedMods/MK2_Expansion/MK2_Utility.cfg [hunk 11]
```diff
@@ -0,0 +1,127 @@
...
+	}
+}
+
+@PART[M2X_Mk2BatteryBank]:FOR[RealismOverhaul]
+{
+    %RSSROConfig = True
+    %rescaleFactor = 1.722222
+    @mass = 0.3
+
+	!RESOURCE[ElectricCharge]
```
Comment (Capkirk123) [inline GameData/RealismOverhaul/RO_SuggestedMods/MK2_Expansion/MK2_Utility.cfg:41]: Missing curly braces

File: GameData/RealismOverhaul/RO_SuggestedMods/Squad/RO_Squad_Command.cfg (hunk 10 above)
Comment (xFluffyDemon) [reply GameData/RealismOverhaul/RO_SuggestedMods/Squad/RO_Squad_Command.cfg:532]: Pls work

**Comment by Capkirk123** [comment]
Looks good to me. Balance is no worse than the rest of the Mk2 parts, at least.

**Comment by siimav** [comment]
The files need a tabification pass though. Indenting looks very inconsistent right now.

**Comment by siimav** [comment]
Indenting looks a lot better now but there's still a few places where it's off. You can easily spot those by scrolling through the "Files changed" tab in GH web UI.

**Comment by siimav** [comment]
I think it needs a 3rd time as well. :D

**Pull request merged**